app/scripts/application.js

```diff
@@ -0,0 +1,24 @@
+import AppConfig from 'scripts/config';
+import LinkOverride from 'scripts/overrides/link';
+
+let App = new Marionette.Application();
+
+App.history = function() {
```

Comment:
in this case context doesn't make any difference, does it? Using fat arrow here will help with readability. Compare:

```js
App.history = function() {}
```

vs

```js
App.history = () => {}
```

Cleaner syntax and less typing.

IMO, we should favor using arrow functions in our es6 codebases for the same reasons as specified in [Airbnb JS styleguide](https://github.com/airbnb/javascript#8.1):

> Why? It creates a version of the function that executes in the context of `this`, which is usually what you want, and is a more concise syntax.